src/AutoRest.CSharp.V3/AutoRest/Plugins/CSharpGen.cs

```diff
@@ -96,7 +96,7 @@ private KnownMediaType[] GetMediaTypes(CodeModel codeModel)
                         types.Add(bodyRequest.KnownMediaType);
                     }
 
-                    foreach (var response in operation.Responses!)
+                    foreach (var response in operation.Responses ?? Array.Empty<ServiceResponse>())
```

Comment:
I was thinking recently if we should start had writing the input model and make it nice to consume by fixing all this nullable noise.

Reply:
The nullability for it is added on. Meaning, it can be turned off. However, then you never know what is expected to be nullable or not. In this case, a valid operation can have no responses.

Reply:
We can always initialize property to `Array.Empty()`

Reply:
I'll make another PR to change nullability assignment to, instead, initialize with empty collections/values.